kalkulator.py

```diff
@@ -5,7 +5,12 @@ def dodaj():
     wynik = a + b
     return wynik
 
+def odejmij(a,b):
+    return a - b
```

Comment:
Moim zdaniem tutaj lepiej rozbić to na dwie linijki, wprowadzając zmienną "wynik"